### PR TITLE
[Administration & Dashboard] Collections updates for 5-2 by AN.

### PR DESCRIPTION
**Pull request by Aishbn**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- After: https://<branch>--prisma-cloud-docs--hlxsites.hlx.page/
- Worker: https://prisma-cloud-docs-production.adobeaem.workers.dev/?branch=<branch>

### CODE REVIEW

File: docs/en/enterprise-edition/content-collections/dashboards/dashboards-discovery-exposure-management.adoc [hunk 1]
```diff
@@ -36,23 +39,38 @@ Displays a complete view of your cloud assets discovered across your attack surf
...
 
-In addition to conversion options, the *Inspect Unmanaged Assets* column lists the scanned asset findings, enabling you to review the exposed assets. Select a link to view the side panel. The panel includes a comprehensive list of assets, risk types, and associated cloud providers. Further exploration of each asset provides detailed information such as asset *Overview*, *Exposed Services*, identified *Findings*, and *Network Flow Log* visualizations.
+In addition to conversion options, the *Inspect Unmanaged Assets* column lists the scanned asset findings, enabling you to review the exposed assets. Select a link to view the side panel. The panel includes a comprehensive list of assets, risk types, and associated cloud providers. Further exploration of each asset provides detailed information as follows:
+
+* *Overview—* This is the default tab providing a comprehensive overview of the selected asset, including details such as server name, resource type, expiry date, and so on.
```
Comment (jenjoe22) [inline docs/en/enterprise-edition/content-collections/dashboards/dashboards-discovery-exposure-management.adoc:68]: I'd suggest using etc here

This is the default tab providing a comprehensive overview of the selected asset, including details such as server name, resource type, expiry date etc.

File: docs/en/enterprise-edition/content-collections/dashboards/dashboards-discovery-exposure-management.adoc [hunk 2]
```diff
@@ -36,23 +39,38 @@ Displays a complete view of your cloud assets discovered across your attack surf
...
+* *Exposed Service—* Displays the list of exposed services that are running on the selected asset, such as HTTP servers and other relevant services. Select a listed service to access granular information on the *IPs* and *Certificates* associated with it.
+* *Vulnerabilities*: Displays Common Vulnerabilities and Exposures (CVE) discovered on the asset. The severity levels of the CVEs are categorized based on the Common Vulnerability Scoring System (CVSS) scores. Also, the *Impacted Distros* section provides more details on the affected versions of a particular package.
```
Comment (jenjoe22) [inline docs/en/enterprise-edition/content-collections/dashboards/dashboards-discovery-exposure-management.adoc:70]: Suggest minor rewrite:

The *Impacted Distros* section, also provides more details on the affected versions of a particular package.

File: docs/en/enterprise-edition/content-collections/dashboards/dashboards-discovery-exposure-management.adoc [hunk 3]
```diff
@@ -65,17 +83,18 @@ While you *Inspect Unmanaged Assets*, you can choose to snooze your known intern
...
+The *Convert* button is enabled only for accounts associated with the parent account or organizational unit (OU) accounts on Prisma Cloud. Selecting the *Convert* button will onboard the assets to Prisma Cloud. Once the unmanaged assets convert to managed assets, Prisma Cloud will monitor these assets for vulnerabilities, misconfigurations, compliance issues, and so on.
 
+In cases where such mapping is not feasible, you can use the *Inspect* button to identify issues related to exposed assets and *Send email* to your inbox containing the summary of the exposed assets. Selecting the *Inspect* button opens a side panel. The panel includes a comprehensive list of exposed assets, risk categories, and associated cloud providers. Further exploration of each asset provides detailed information such as asset *Overview*, *Exposed Services*, *Vulnerabilities*, *Findings*, and *Network Flow Log* visualizations.
```
Comment (jenjoe22) [inline docs/en/enterprise-edition/content-collections/dashboards/dashboards-discovery-exposure-management.adoc:95]: Suggest the following rewrite:

In cases where such mapping is not feasible, use the *Inspect* button to identify issues related to exposed assets and *Send email* to your inbox containing the summary of the exposed assets. Select *Inspect* to view a side panel, with a comprehensive list of exposed assets, risk categories, and associated cloud providers. Further exploration of each asset provides detailed information such as asset *Overview*, *Exposed Services*, *Vulnerabilities*, *Findings*, and *Network Flow Log* visualizations.

File: docs/en/enterprise-edition/content-collections/dashboards/dashboards-discovery-exposure-management.adoc (hunk 2 above)
Comment (Aishbn) [reply docs/en/enterprise-edition/content-collections/dashboards/dashboards-discovery-exposure-management.adoc:70]: Done

**Pull request closed without merging**